tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -621,7 +623,9 @@ public function testGetColumnsReservedTableName()
               ->addColumn('column17', 'point')
               ->addColumn('column18', 'linestring')
               ->addColumn('column19', 'polygon')
-              ->addColumn('column20', 'uuid');
+              ->addColumn('column20', 'uuid')
+              ->addColumn('column21', 'set', array('values' => "one, two"))
```

Comment:
this example will be slightly broken because of the issue with spaces in a comma separated list